### ISSUE TITLE
Favorites and History exports raise a storage error

### ISSUE THREAD

**Issue by dbcosner** [nelenkov/wwwjdic#111]
When I attempt to export the favorites or history list, I briefly see an error flash:

Export error: /storage/emulated/0/wwwjdic/search-history.csv: ope(...)

Presumably there is more information, but it is cut off.

**Comment by nelenkov** [comment]
Can you check if the app has the external storage permission in settings? What Android version are you using?

**Comment by dbcosner** [comment]
I'm using a Samsung phone with Android 13 and One UI 5.1. I thought it may be a permissions issue, but I do not see an option to enable storage permissions:
![image](https://github.com/user-attachments/assets/2cd33664-02f7-4cf4-83df-eb30a1e767ad)

**Comment by nelenkov** [comment]
Yeah, the original storage permission is not granted any more. Something like this should fix it, but need to update a few more things for a working release: 
https://github.com/nelenkov/wwwjdic/commit/c795afebff0fd6bad41e92c5a9969d0f044b0c40

**Comment by dbcosner** [comment]
I appreciate the response! Let me know if you'd like me to test a pre-release build. I use this app constantly and I'm glad to help out.

**Comment by nelenkov** [comment]
Latest debug version is here if you want to try it out: https://github.com/nelenkov/wwwjdic/releases/tag/latest

You can't install on top of the release version and currently quite a few things are broken, hopefully I'll get the time to stabilize it.

**Comment by dbcosner** [comment]
Thanks! I'll install the debug version over the weekend and get back to you.

**Comment by dbcosner** [comment]
Got this error when attempting to install via ADB: `adb: failed to install wwwjdic-android-2.6.0-370a37f.apk: Failure [INSTALL_FAILED_CONFLICTING_PROVIDER: Scanning Failed.: Can't install because provider name null.fileprovider (in package org.nick.wwwjdic) is already used by com.mcdonalds.app]`

I tried just transferring the file to the phone, but it does not give me the option to install when I clicked the APK from the built-in file explorer.

Update: I was able to install via ADB by deleting the McDonald's app. I was able to export my history!

**Comment by nelenkov** [comment]
Thanks for testing! Latest push should fix that. BTW, it appears that the McDonalds app is also misconfigured for some reason.

**Comment by nelenkov** [comment]
Fix released on Play Store: https://play.google.com/store/apps/details?id=net.fafla.wwwjdic
